packages/patterns/src/components/partial-syncing-controls.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { nanoid } from 'nanoid';
+
+/**
+ * WordPress dependencies
+ */
+import { BaseControl, CheckboxControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+
+function PartialSyncingControls( { attributes, setAttributes } ) {
+	// Only the `content` attribute of the paragraph block is currently supported.
+	const attributeName = 'content';
```

Comment:
I guess we need to think about how to control which blocks/attributes are supported. It might be as simple as some config:
```js
`PARTIAL_SYNC_SUPPORT = {
    'core/paragraph': { attributes: [ 'content' ] }
 }
 ```